src/@next/Banner/FixedBannerStyle.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
```

Comment:
Do we need  it to be `.tsx`?

Reply:
ah no because we don't need any jsx there, okay will rename that to .ts only similar to the other style files